tests/javaRosa.js

```diff
@@ -486,6 +486,7 @@ define([
             var label = util.addQuestion("Trigger", "label"),
                 text2 = util.addQuestion("Text", "text2");
             label.p.labelItext.set('<output value="/data/load-one" />');
+            label.validate("labelItext");
```

Comment:
This is now necessary to register the output value reference because `handleMugRename` no longer does a comprehensive scan of all itext values.